unicodetools/src/test/java/org/unicode/propstest/CompareVersionedProps.java

```diff
@@ -24,6 +25,9 @@
 import org.unicode.tools.emoji.EmojiOrder;
 
 public class CompareVersionedProps {
+    private static IndexUnicodeProperties IUP = IndexUnicodeProperties.make();
+
+    private static final boolean NO_EMOJI = true;
```

Comment:
Is this necessary because of the current not-fully-Unicode-18 state of data?
Will this affect other tools and tests?
When do we need to flip this to false?